extensions/flutter-pub-dev-search/src/index.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  randomId,
+} from "@raycast/api";
+import { useState, useEffect, useRef } from "react";
+import fetch, { AbortError } from "node-fetch";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search for a Dart/Flutter package..." throttle>
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.id} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.name}
+      // subtitle={searchResult.description}
+      // accessoryTitle={searchResult.version}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <CopyToClipboardAction
+              title="Copy Install Command"
+              content={`flutter pub add ${searchResult.name}`}
+              onCopy={(content) => showToast( ToastStyle.Success,`${content} Copied to clipboard!\n Now go and paste the command into your IDE's terminal to add the package.`)}
+            />
+          </ActionPanel.Section>
+          <ActionPanel.Section>
+            <OpenInBrowserAction 
+            title="Open in Browser" 
+            url={searchResult.url} 
+            shortcut={{ modifiers: ["cmd"], key: "." }}
```

Comment:
Here I'd suggest to put this action into the same section as the primary action, so that this will automatically become the secondary action (automatic shortcut is then cmd + enter).

Reply:
How?
by removing the shortcut?

Reply:
You can just remove the sections (there is one implicit section automatically created) and remove the shortcut.

Reply:
Thanks, I have removed the shortcut as you suggested.